packages/components/src/popover/index.js

```diff
@@ -75,7 +75,10 @@ function computeAnchorRect(
 			return getRectangleFromRange( anchorRef );
```

Comment:
Shouldn't the above be changed too? Could potentially check for `.endContainer`.

Reply:
Yes, I was looking at that and I think the range check is susceptible as well. I'll push the change in a moment.